templates/advantage/index.html

```diff
@@ -19,19 +19,19 @@ <h1>Ubuntu Advantage for Infrastructure</h1>
 <section class="p-strip" style="overflow:visible;">
```

Comment:
It needn’t block landing, but: The intro section is so tall that, on my screen, without scrolling I can’t even see whether I’m signed in. This seems suboptimal for a page that people call a “dashboard”. Is there a simple way to reduce the height? It’s using a lot of height for padding and angles.

Reply:
Done and signed off with design.